pkg/pipeline/input/builder/video.go

```diff
@@ -107,7 +107,7 @@ func (v *VideoInput) buildSDKDecoder(p *config.PipelineConfig, src *app.Source,
 	case strings.EqualFold(codec.MimeType, string(types.MimeTypeH264)):
 		if err := src.Element.SetProperty("caps", gst.NewCapsFromString(
 			fmt.Sprintf(
-				"application/x-rtp,media=video,payload=%d,encoding-name=H264,clock-rate=%d",
+				"application/x-rtp,media=video,payload=%d,encoding-name=H264,clock-rate=%d,width=1980,height=1080",
```

Comment:
p.Width/Height?

Reply:
FYI: it's possible for TrackInfo.width to be 0 sometimes. It's a rare condition. we should have some defaults for this if possible.

Also, do the values set here do anything? or is it just to get the downstream caps to be happy?

Reply:
Thanks for catching this. This is test code that made its way in the PR. the h264parse step will extract the actual dimensions.